pl_bolts/models/rl/common/agents.py

```diff
@@ -138,3 +139,44 @@ def __call__(self, states: torch.Tensor, device: str) -> List[int]:
         actions = [np.random.choice(len(prob), p=prob) for prob in prob_np]
 
         return actions
+
+
+class ActorCriticAgent(Agent):
+    """Policy based agent that returns a distribution and an action given an observation"""
+
+    def __init__(self, actor_net: nn.Module, critic_net: nn.Module):
+        self.actor_net = actor_net
+        self.critic_net = critic_net
+
+    @torch.no_grad()
+    def __call__(self, state: torch.Tensor, device: str) -> List[int]:
+        """
+        Takes in the current state and returns the agents policy, a sampled action, log probability
+        of the action, and value of the given state
+        Args:
+            states: current state of the environment
+            device: the device used for the current batch
+        Returns:
+            torch dsitribution and randomly sampled action
+        """
+
+        state = state.to(device=device)
+
+        pi, actions = self.actor_net(state)
+        log_p = self.get_log_prob(pi, actions)
```

Comment:
redundant creating this agent for this implementation as I can do the same in `model.forward`. but doing it like this keeps up with the other implementation.   
                                                                                                                                       
let me know your thoughts on removing this and just using model.forward in the training loop